client/src/templates/Challenges/projects/SolutionForm.tsx

```diff
@@ -39,9 +50,9 @@ export class SolutionForm extends Component {
         this.props.onSubmit({ isShouldCompletionModalOpen: false });
       }
     }
-  }
+  };
 
-  render() {
+  render(): JSX.Element {
     const { isSubmitting, challengeType, description, t } = this.props;
```

Comment:
```suggestion
    const { challengeType, description, t } = this.props;
```
There instance where `isSubmitting` is used and that can just be simplified to the `false` outcome.

Reply:
Where is it used actually? Linting is saying it's never used.